raiden/storage/serialization/serializer.py

```diff
@@ -44,16 +48,31 @@ def serialize(obj):
         # Default, in case this is not a dataclass
         data = obj
         if is_dataclass(obj):
-            schema = SchemaCache.get_or_create_schema(obj.__class__)
-            data = schema.dump(obj)
+            try:
+                schema = SchemaCache.get_or_create_schema(obj.__class__)
+                data = schema.dump(obj)
+            except (TypeError, ValidationError) as ex:
```

Comment:
what can raise `TypeError`?

Reply:
I have the impression `SchemaCache.get_or_create_schema` doesn't raise any of the mentioned exceptions, could you please move it out of the `try` block?